code/src/App.js

```diff
@@ -1,9 +1,116 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
+import ThoughtsList from 'components/ThoughtsList';
+import ThoughtForm from 'components/ThoughtForm';
+
 
 export const App = () => {
+  const [counter, setCounter] = useState(0);
+  const [newThought, setNewThought] = useState('');
+  const [thoughts, setThoughts] = useState([]);
+  const [loading, setLoading] = useState(false);
+  // useEffect with empty array [] call your functions on application start
+  
+  
+  const fetchData = () => {
+    setLoading(true);
+    fetch('https://happy-thoughts-technigo.herokuapp.com/thoughts')
+      .then((response) => response.json())
+      .then((Data) => setThoughts(Data)) 
+      .catch((error) => console.error(error))
+      .then(console.log('works'))
+      .finally(() => setLoading(false));
+  }
+  //get 20 thoughts from API
+  useEffect(() => {
+    fetchData();
+  }, []); 
+
+
+  // will trigger first when app starts, and every time the variable in the dependency array changes
+  useEffect(() => {
+    //window.alert(`the current counter is ${counter}`);
+    }, [counter]);
+  
+
+  const onNewThoughtChange = (event) => {
+    setNewThought(event.target.value);
+  }
+
+const handleHeartCounter = (_ID) => {
+  const options = {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json',
+    },
+  };
+
+//  get likes 
+  fetch( `https://happy-thoughts-technigo.herokuapp.com/thoughts/${_ID}/like`, options)
+    then((response) => response.json())
```

Comment:
Could the problem with the like fetching be as easy as adding a dot before the "then"?  =>  .then